pkg/gits/operations/pull_request_op.go

```diff
@@ -230,8 +230,10 @@ func AddDependencyMatrixUpdatePaths(upstreamDependencyAsset *gits.GitReleaseAsse
 	if err != nil {
 		return nil, errors.Wrapf(err, "retrieving dependency updates from %s", upstreamDependencyAsset.BrowserDownloadURL)
 	}
-	err = yaml.Unmarshal(b.Bytes(), &upstreamUpdates)
+	bytes := b.Bytes()
+	err = yaml.Unmarshal(bytes, &upstreamUpdates)
 	if err != nil {
+		log.Logger().Errorf("unable to unmarshall from %s", string(bytes))
```

Comment:
would you not put this into the error created in `errors.Wrapf`?  Or if it is really debug log, should it not be `Debugf`?